docs/initializing.rst

```diff
@@ -555,6 +556,42 @@ point into the system. It will handle the setup and initialization of the config
 and start up a Toolkit session once all the required pieces have been initialized and set up.
 
 
+Distributing application code from your site
```

Comment:
Here's what this section actually looks like.
![screen shot 2018-09-19 at 15 57 44](https://user-images.githubusercontent.com/8126447/45777945-342d5c80-bc25-11e8-8142-4fdb39313fab.png)
![screen shot 2018-09-19 at 15 57 52](https://user-images.githubusercontent.com/8126447/45777955-37c0e380-bc25-11e8-93a5-76cb38d197a2.png)

Reply:
About the "Click here to learn more" bit: once we have actual hook documentation for tk-core, we can update this link to the proper link in the doc.